src/main/java/training/endava/skeleton/Classes/PersonRepository.java

```diff
@@ -0,0 +1,72 @@
+package training.endava.skeleton.Classes;
+
+import training.endava.skeleton.Exceptions.IncorectRuntimeException;
+import training.endava.skeleton.Interfaces.SkeletonRepository;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Optional;
+
+public class PersonRepository implements SkeletonRepository<Person, Integer> {
+
+    List<Person> personList;
+
+    public PersonRepository(List<Person> personList) {
+        this.personList = personList;
+    }
+
+    @Override
```

Comment:
You should use MOCK_DB in PersonRepository for all the methods (CRUD operations). Please review them.